packages-exp/functions-exp/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@firebase/functions-exp",
+  "version": "0.0.800",
+  "description": "",
+  "private": true,
+  "author": "Firebase <firebase-support@google.com> (https://firebase.google.com/)",
+  "main": "dist/index.node.cjs.js",
+  "browser": "dist/index.esm.js",
+  "module": "dist/index.esm.js",
+  "esm2017": "dist/index.esm2017.js",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "lint": "eslint -c .eslintrc.js '**/*.ts' --ignore-path '../../.gitignore'",
+    "lint:fix": "eslint --fix -c .eslintrc.js '**/*.ts' --ignore-path '../../.gitignore'",
+    "build": "rollup -c && yarn api-report",
+    "build:deps": "lerna run --scope @firebase/'{app-exp,functions-exp}' --include-dependencies build",
+    "dev": "rollup -c -w",
+    "test": "yarn type-check && run-p lint test:browser test:node",
+    "test:ci": "node ../../scripts/run_tests_in_ci.js",
+    "type-check": "tsc -p . --noEmit",
+    "test:browser": "karma start --single-run",
+    "test:browser:debug": "karma start --browsers=Chrome --auto-watch",
+    "test:node": "TS_NODE_COMPILER_OPTIONS='{\"module\":\"commonjs\"}' nyc --reporter lcovonly -- mocha 'src/{,!(browser)/**/}*.test.ts' --file src/index.node.ts --config ../../config/mocharc.node.js",
+    "test:emulator": "env FIREBASE_FUNCTIONS_EMULATOR_ORIGIN=http://localhost:5005 run-p test:node",
+    "prepare": "yarn build",
+    "api-report": "api-extractor run --local --verbose",
+    "predoc": "node ../../scripts/exp/remove-exp.js temp",
+    "doc": "api-documenter markdown --input temp --output docs",
+    "build:doc": "yarn build && yarn doc"
+  },
+  "license": "Apache-2.0",
+  "peerDependencies": {
+    "@firebase/app-exp-types": "0.x"
+  },
+  "devDependencies": {
+    "@firebase/app-exp": "0.x",
```

Comment:
`"@firebase/app-exp": "0.x"` still needs to be in peerDependencies.
Does workspace linking work correctly with `0.x`? I thought you'd need to use the exact version.